GCDWebServer.podspec

```diff
@@ -35,7 +35,7 @@ Pod::Spec.new do |s|
   
   s.subspec "CocoaLumberjack" do |cs|
     cs.dependency 'GCDWebServer/Core'
```

Comment:
Cocoalumberjack did the switch to swift 3 with the version 3.0.0. 
Some other projects do keep a separate branch for swift 3, so for a limited time u'll have support for both and insome months they'll probably drop swift 2